idd/Energy+.idd.in

```diff
@@ -5,6 +5,14 @@
 ! The IDD defines the syntax and data model for each type of input "Object."
 ! Lines in EnergyPlus input files (and IDD) are limited to 500 characters.
 !
+!
+! Note on IDD Processing/Usage
```

Comment:
This entire block of comments also appear in the I/O Reference. I would just copy from the IDD to update the entire block (which may be out of date in other ways0.

Reply:
Fixed--see next commit.